packages/react-scripts/template/README.md

```diff
@@ -430,8 +429,8 @@ Then we can change `start` and `build` scripts to include the CSS preprocessor c
 
 ```diff
    "scripts": {
-     "build-css": "node-sass src/ -o src/",
-     "watch-css": "npm run build-css && node-sass src/ -o src/ --watch --recursive",
+     "build-css": "node-sass-chokidar src/ -o src/",
+     "watch-css": "npm run build-css && node-sass-chokidar src/ -o src/ --watch --recursive",
```

Comment:
Can we hide css compiled files in "npm start"? 
I saw many other sources, they can run with "npm start" without compiling scss files. Imagine that my folders have many new css files, it looks ugly. Thanks for any help!